data/curated/american_idol/ratings.md

```diff
@@ -0,0 +1,19 @@
+|variable                |class     |description             |
+|:-----------------------|:---------|:-----------------------|
+|season                  |double    |Season                  |
+|show_number             |double    |Episode number in season     |
+|episode                 |character |Episode name                 |
+|airdate                 |character |Date episode aired          |
+|18_49_rating_share      |character |Percentage of adults aged 18-49 estimated to have watched the episode (Nielsen TV ratings). |
+|viewers_in_millions     |double    |Number (in millions) that watched the episode |
+|timeslot_et             |character |Episode timeslot in Eastern Time     |
+|dvr_18_49               |character |Percentage of adults aged 18-19 estimated to have watched the episode on DVR     |
+|dvr_viewers_millions    |character |Number (in millions) that watched the episode on DVR  |
+|total_18_49             |character |Total percentage of adults aged 18-49 estimated to have watched the episode    |
+|total_viewers_millions  |character |Total number of viewers (in millions). |
+|weekrank                |character |Ranking of episode performance by season            |
+|ref                     |logical   |Reference                    |
+|share                   |character |share                   |
```

Comment:
```suggestion
|share                   |character |share  (unused)   |
```

Technically this column should probably be dropped (it appears to always be `NA`) but meh real data has unused columns sometimes.